src/app.ts

```diff
@@ -146,17 +147,15 @@ ${query.text}`
         )
       );
 
-      const ctype = driver.columnType;
```

Comment:
we can do

```ts
const ctype = driver.columnType.bind(driver)
```

instead to preserve `this` in `columnType()`, but passing the driver directly is easier to understand.